### PR TITLE
netlink attribute type flags

### PR DESCRIPTION
**Pull request by teg**
Respect the netlink attribute type flags.

### CODE REVIEW

**Comment by dvdhrm** [comment]
You said you'll rework this to use "struct netlink_container", right? In that case, I think we can close this PR.

File: src/libsystemd/sd-netlink/netlink-message.c [hunk 1]
```diff
@@ -739,7 +798,7 @@ int sd_netlink_message_enter_container(sd_netlink_message *m, unsigned short typ
         } else
                 return -EINVAL;
 
-        r = rtnl_message_read_internal(m, type, &container);
+        r = rtnl_message_read_internal(m, type, &container, &flags);
```
Comment (zonque) [inline src/libsystemd/sd-netlink/netlink-message.c:801]: Instead of catching `flags` into a throw-away variable, you could just use `NULL` here, right?

Comment (teg) [inline src/libsystemd/sd-netlink/netlink-message.c:801]: Indeed, hat was a left-over from a previous rework. Thanks!

**Comment by dvdhrm** [comment]
As discussed privately: Apart from the misleading commit-msg in the second patch, looks all good to me.

**Comment by teg** [comment]
This is deprecated by PR #398

**Pull request closed without merging**